prime-router/src/test/kotlin/azure/SubmissionFunctionTests.kt

```diff
@@ -80,7 +80,8 @@ class SubmissionFunctionTests : Logging {
         )
```

Comment:
ignore changes to source files. I just moved code around so it would be updated and trigger actions.